<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 25 10pt &quot;微软雅黑 Light&quot;;
background-image: url(:/new/prefix1/123.jpg);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Forest &amp;amp; BinaryTree&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/123.jpg);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;BinaryTree&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="RootEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FatherL">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LChild">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FatherR">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="RChild">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize Root</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertLChild">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert LChild</string>
    </property>
   </widget>
   <widget class="QPushButton" name="InsertRChild">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert RChild</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTreeShow">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualition</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Change">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/new/prefix1/123.jpg);</string>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Forest&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="FatherInsert">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Root1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Root2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Node</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ChildInsert">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Link Root</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FatherDelete">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ChildDelete">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Node</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>380</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>431</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>441</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
